plugins/discourse-lazy-videos/config/settings.yml

```
@@ -0,0 +1,17 @@
+plugins:
+  lazy_videos_enabled:
+    default: true
+    client: true
+    hidden: false
```

I think we can keep this setting hidden. 

```suggestion
    hidden: true
```